example/README.md

```diff
@@ -1,114 +1,176 @@
 
-    i.e.
-        ~/<repos directory>/RackHD/
+Clone RackHD repo to your local git directory.
 
+    $ git clone https://github.com/RackHD/RackHD
+    $ cd RackHD
 
-  Within the example directory, create config and run the setup command:
 
-    $ cd ~/<repos directory>/RackHD/example/config/
+Change into the directory `example`, create config and run the setup command:
 
+    $ pushd example/config/
     $ cp ./monorail_rack.cfg.example ./monorail_rack.cfg
+    $ popd
 
-    Edits can be made to this new file to adjust the number of pxe clients created.
-    Please see below for more information on the configuration file.
-
-    $ cd ~/<repos directory>/RackHD/example/bin/
+Edits can be made to this new file to adjust the number of pxe clients created.
 
+    $ pushd bin/
     $ ./monorail_rack
 
-  Copy local basic static files to common directory:
+Now ssh into the RackHD server and start the services
 
-    $ cp ~/<static files directory>/* ~/<repos directory>/RackHD/on-http/static/http/common/
+    $ vagrant ssh
+    $ sudo nf start
 
-  Now ssh into the monorail server:
+## TESTING
 
-    $ vagrant ssh dev
+Once you've started the services, the RackHD API will be available on your local
+machine through port 9090. For example, you should be able to view the RackHD
+API documentation that's set up with the service at http://localhost:9090/docs.
 
-  Bring up all monorail services:
+You can also interact with the APIs using curl from the command line of your
+local machine.
 
-    $ sudo nf start
-      or $ sudo nf start [graph,http,dhcp,tftp,syslog]
+To view the list of nodes that has been discovered:
+    $ curl http://localhost:9090/api/1.1/nodes | python -m json.tool
 
-    Now that the services are running we can begin powering on pxe clients and watch them boot.
+View the list of catalogs logged into RackHD:
+    $ curl http://localhost:9090/api/1.1/catalogs | python -m json.tool
 
+(both of these should result in empty lists in a brand new installation)
 
-  Provision an existing monorail server:
+### Install a default workflow for Virtualbox VMs and a SKUs definition
 
-    $ vagrant provision
+This example includes a workflow that we'll use when we identify a "virtualbox"
+SKU with RackHD. We'll load it into our library of workflows:
 
-## CONFIGURATION FILE
+    cd ~/src/rackhd/example
+    # make sure you're in the example directory to reference the sample JSON correctly
 
-```
-# monorail_rack.cfg
-# used to customize default deployment
-# edit $pxe_count to change the amount of virtualbox PXE-booting clients are created when running
-# the monorail_rack setup script.
+    curl -H "Content-Type: application/json" \
+    -X PUT --data @samples/virtualbox_install_coreos.json \
+    http://localhost:9090/api/1.1/workflows
 
-# deployment variables
-pxe_count=1
-```
+We also include the sku definition. RackHD sku support includes a mechanism that
+will run a workflow when a SKU is identified. In this case, the example uses the
```

Comment:
How's something like this:

This example includes a workflow that we'll use when we identify a "virtualbox"
SKU with RackHD. This workflow sets up no-op out of band management settings
for a demo and triggers an installation of CoreOS as a default flow to run
once the "virtualbox" SKU has been identified. We'll load it into our library
of workflows:

    cd ~/src/rackhd/example
    # make sure you're in the example directory to reference the sample JSON correctly

    curl -H "Content-Type: application/json" \
    -X PUT --data @samples/virtualbox_install_coreos.json \
    http://localhost:9090/api/1.1/workflows

To enable that workflow, we also need to include a SKU definition that includes
the option of another workflow to run once the SKU has been identified. This
takes advantage of the discoverySKU workflow, which will attempt to identify a
SKU and run another workflow if specified.